db/seeds.rb

```diff
@@ -8,6 +8,7 @@
                     fetch(Rails.env, {})
 
 service_providers.each do |issuer, config|
+  next if Figaro.env.chef_env == 'prod' && config['live'] != 'true'
```

Comment:
in all our AWS envs, the `RAILS_ENV == production` so we need some way to mark which SPs should only be deployed in the live `prod` AWS env.